portfolio/src/main/webapp/style.css

```diff
@@ -5,7 +5,14 @@
 }
 
 #greeting-container {
-  margin-top: 20px;
+        margin-top: 20px;
+	font-family:"Courier New", Courier, monospace;
+	font-size: 30px;
+	color: white;
+	margin-left: auto;
+	margin-right: auto;
+	width: 80vw;
+	text-align: center;
```

Comment:
Nit, and I realize this wasn't being followed in the starter code, but css declarations should be in alphabetical order. Don't feel the need to fix every occurrence now though

From the style guide:
Alphabetize declarations.

Put declarations in alphabetical order in order to achieve consistent code in a way that is easy to remember and maintain.

Ignore vendor-specific prefixes for sorting purposes. However, multiple vendor-specific prefixes for a certain CSS property should be kept sorted (e.g. -moz prefix comes before -webkit).
ex:
background: fuchsia;
border: 1px solid;
-moz-border-radius: 4px;
-webkit-border-radius: 4px;
border-radius: 4px;
color: black;
text-align: center;
text-indent: 2em;